docs/lmsIntegrationInstructor.md

```diff
@@ -1,10 +1,12 @@
 # LMS integration for instructors
 
-Integrating Learning Management Systems (LMSes) with PrairieLearn uses [LTI 1.3](https://www.1edtech.org/standards/lti) technology to connect LMS courses with PrairieLearn course instances. This allows for programmatic assignment linking and grade passback.
+Integrating Learning Management Systems (LMSes) with PrairieLearn uses [LTI 1.3](https://www.1edtech.org/standards/lti) technology to connect LMS courses with PrairieLearn course instances. This allows for programmatic assignment linking and grade syncing.
```

Comment:
"passback" -> "syncing" feels slightly nicer to me.

Reply:
Agreed.

Reply:
Or just simplify this whole paragraph:
```suggestion
PrairieLearn can connect to Learning Management Systems (LMSes) using [LTI 1.3](https://www.1edtech.org/standards/lti) technology. This allows for assignment linking and grade syncing.
```

Reply:
Love it, this is much simpler and still gets the point across well.